src/app/components/Unit.tsx

```diff
@@ -34,6 +40,7 @@ class Unit extends React.Component<Props> {
       denomination,
       isFiatPrimary,
       isNoFiat,
+      chain
```

Comment:
I believe the the style is to keep a comma at the end of the last element.

```suggestion
      chain,
```